torch/csrc/jit/passes/erase_number_types.cpp

```diff
@@ -39,8 +39,10 @@ static void EraseNumberTypesOnBlock(Block* block) {
       } break;
       case aten::Bool:
       case aten::Float:
+      case aten::Complex:
```

Comment:
onnx only supports tensors, so this is used to turn all out number types into tensors

Reply:
ONNX does not support complex tensors yet. Could you leave out this change and have given onnx conversion properly error out with `aten::Complex` not supported?

Reply:
do you know if `createNumToTensor` was only added to support onnx?

Reply:
thanks! yea i think currently only onnx conversion is using that.